docs/source/tutorial/detection.rst

```diff
@@ -146,14 +146,14 @@ Inference on these models runs smoothly by downloading necessary pre-trained wei
     :scale: 60%
     :align: center
 
-The above example puts together functinality of detection link.
+The above example puts together functionality of detection link.
 It instantiates SSD300 model with weights trained on VOC 2007 and VOC 2012 datasets.
 The model runs prediction using :meth:`~chainercv.links.model.ssd.SSD.predict`, and the outputs are visualized using
 :func:`~chainercv.visualizations.vis_bbox`.
 Note that in this case, confidence scores are visualized together with other data.
 
 Many detection algorithms post-process bounding box proposals calculated from the output of neural networks by removing unnecessary ones.
-In the case of Faster R-CNN and SSD, bounding boxes with low confidence are suppressed and overlapping boxes are removed using non-maximum suppression.
+In the case of Faster R-CNN and SSD, non-maximum suppression suppresses bounding boxes with low confidence and removes overlapping boxes.
```

Comment:
`suppression suppresses` sounds redundant and non-maximum suppression does not remove bounding boxes with low confidence.
How about `Faster R-CNN and SSD use non-maximum suppression to remove overlapping bounding boxes. They also remove bounding boxes with low confidence scores`?
Perhaps, `Faster R-CNN and SSD use non-maximum suppression to remove overlapping bounding boxes.` is enough.